src/features/bedrijf/bedrijf-zoeken/BedrijvenOverzichtRow.vue

```diff
@@ -1,168 +1,218 @@
+            ", ",
+          )
+        }}
       </template>
     </td>
     <td>
-      <div class="skeleton" v-if="matchingKlant.loading || bedrijf.loading" />
-
-      <template v-if="matchingKlant.success && matchingKlant.data">
-        <router-link
-          :title="`Details ${naam}`"
-          :to="getKlantUrl(matchingKlant.data)"
-          @click="setCache(matchingKlant.data, bedrijf.data)"
-        />
-      </template>
-      <button
-        v-else-if="bedrijf.data && bedrijfIdentifier"
-        type="button"
-        title="Aanmaken"
-        @click="navigate(bedrijf.data, bedrijfIdentifier)"
-      />
+      <button v-if="item" type="button" title="Details" @click="navigate()" />
     </td>
   </tr>
 </template>
 <script lang="ts" setup>
-import { computed, watchEffect } from "vue";
-
-import { useKlantByBedrijfIdentifier } from "./use-klant-by-bedrijf-identifier";
-import type { Bedrijf, BedrijfIdentifier } from "@/services/kvk";
+import { type Bedrijf } from "@/services/kvk";
 import { useRouter } from "vue-router";
-import { mutate } from "swrv";
-import { ensureKlantForBedrijfIdentifier } from "./ensure-klant-for-bedrijf-identifier";
-import type { Klant } from "@/services/openklant/types";
-import type { KlantBedrijfIdentifier } from "@/services/openklant2";
+import { useSystemen } from "@/services/environment/fetch-systemen";
+import {
+  useContactmomentStore,
+  type ContactmomentKlant,
+} from "@/stores/contactmoment";
+import type { KlantIdentificator } from "@/features/contact/types";
+import { useLoader } from "@/services";
+import {
+  fetchKlantByKlantIdentificatorOk,
+  fetchKlantFromNonDefaultSystems,
+  heeftContactgegevens,
+} from "@/features/klant/klant-details/fetch-klant";
 
 const props = defineProps<{
-  item: Bedrijf | Klant;
+  item: Bedrijf;
   autoNavigate?: boolean;
 }>();
 
-// const matchingBedrijf = useBedrijfByIdentifier(() => {
-// wordt niet meer gebruikt, alleen relevant als we een klant hebben en kvkv gegevens erbij willen zoeken/
-// maar dat was alleen relevant toen een klant ook uit openklant gevonden kon worden adhv telefoonnumer en email, maar dat is momenteel niet meer mogelijk
-//   if (props.item._typeOfKlant === "bedrijf") return undefined;
-//   const { vestigingsnummer, rsin } = props.item;
-//   if (vestigingsnummer)
-//     return {
-//       vestigingsnummer,
-//     };
-//   if (rsin)
-//     return {
-//       rsin,
-//     };
-// });
-
-const matchingKlant = useKlantByBedrijfIdentifier(() => {
-  if (props.item._typeOfKlant === "klant") return undefined;
-
-  const { vestigingsnummer, kvkNummer } = props.item;
-
-  if (vestigingsnummer && kvkNummer)
-    return {
-      vestigingsnummer,
-      kvkNummer,
-    };
+const systemen = useSystemen();
+const contactmomentStore = useContactmomentStore();
+
+//Please note, the implementation for bedrijven is a litle more complex than for personen.
```

Comment:
misschien kunnen we overleggen of de contactgegevens weg mogen uit het overzicht? dat zou een hoop complexiteit schelen denk ik

Reply:
is een story voor gemaakt